### ISSUE TITLE
Formatting is way off now compared to before

### ISSUE THREAD

**Issue by asdf337** [poeaffix/poeaffix.github.io#7]
The formatting of how the rolls are so bunched together and long now compared to before, did you add something on the right side and force the whole page format to elongate?

**Comment by ImaginaryDevelopment** [comment]
no, there's a lot more information per mod item though, perhaps that is causing it?

**Comment by asdf337** [comment]
Its looking very bunched up right now, maybe it could be an idea to up the
size sideways, just a thought, my friend was saying the same thing, I tried
adjusting the size in firefox but nothing happened.
